src/site-constants.js

```diff
@@ -14,5 +14,6 @@
 // the SSR part in node.js we won't have a proper location.
 const urlRoot = 'https://reactjs.org';
 const version = '16.0.0';
+const babelURL = '//unpkg.com/babel-standalone@6.26.0/babel.min.js';
 
-export {urlRoot, version};
+export {urlRoot, version, babelURL,};
```

Comment:
Run `yarn prettier` to fix this formatting ^ (the trailing comma should be removed)